modules/globebrowsing/scripts/layer_support.lua

```diff
@@ -217,11 +218,16 @@ openspace.globebrowsing.parseInfoFile = function (file)
     local ColorFile = rawget(_G, "ColorFile")
     local HeightFile = rawget(_G, "HeightFile")
     local Location = rawget(_G, "Location")
+    local ZIndex = rawget(_G, "ZIndex")
 
     -- Now we can start
     local name = Name or Identifier
     local identifier = Identifier
+    local zIndex = ZIndex
 
+    if zIndex == "" then
+        zIndex = 999
```

Comment:
I don't think this is needed. When adding a layer without specifying the ZIndex it will be put right after the latest layer that has been loaded, similar to how it worked before. Therefore I don't think we need a default value.